docs/jsonapi-spec.md

```diff
@@ -473,10 +473,38 @@ Commands are defined using the BNF-like syntax, with samples presented using a [
 
 ### countDocuments Command
 
+Returns the count of documents that match the query for a collection or view.
+
+*Sample:*
+
+```json
+db.orders.countDocuments({})
+```
+
 #### countDocuments Command Options
 
 The `countDocuments` command does not support any options.
 
+#### countDocuments TODO considersations
+
+re: https://datastax.jira.com/browse/C2-2491
+
+* What are the limitations on the filters?
+
+* What are the performance problems?
+
+See the IMPORTANT note in https://www.mongodb.com/docs/manual/reference/method/db.collection.countDocuments/#transactions.
```

Comment:
These comment lines need to be removed, not sure if needs to be in spec. May be start a new discussion or issue for these comment in github.

Reply:
done in next commit